src/java.desktop/share/classes/java/awt/image/BufferedImage.java

```diff
@@ -1549,20 +1555,25 @@ public void addTileObserver (TileObserver to) {
    * Removes a tile observer.  If the observer was not registered,
    * nothing happens.  If the observer was registered for multiple
    * notifications, it is now registered for one fewer notification.
+   * <p>
+   * The default implementation ignores the given observer,
+   * since {@link #addTileObserver(TileObserver)} adds none.
+   *
    * @param to the specified {@code TileObserver}.
    */
     public void removeTileObserver (TileObserver to) {
     }
 
     /**
      * Returns whether or not a tile is currently checked out for writing.
+     *
```

Comment:
I think we need a few words here too -
"The only tile in a {@code BufferedImage} is at (0,0) and it is always writable, so calling this method with (0,0)
will always return {@code true}, and any other coordinate will cause an exception to be thrown".